.template/spec/base/config/environments/test_spec.rb

```diff
@@ -11,6 +11,10 @@
     expect(subject).to contain(bullet_config)
   end
 
+  it 'eager load on CI' do
+    expect(subject).to contain('config.eager_load = ENV["CI"].present?')
+  end
```

Comment:
Just to confirm, the `config.eager_load = ENV["CI"].present?`: Is it already implemented? (I don't see its implementation in the code changes of this PR :D )

Reply:
Yeah, Rails 7+ generator has that by default